TrafficCapture/dockerSolution/build.gradle

```diff
@@ -37,12 +35,19 @@ dockerFilesForExternalServices.each { projectName, dockerImageName ->
     def escapedProjectName = projectName;
     task("buildDockerImage_${escapedProjectName}", type: DockerBuildImage) {
         if (escapedProjectName == "migrationConsole") {
-            copyMigrationConsoleArtifactsFromProjectsToProjectsDockerStaging(project as Project,
-                    List.of(project(":TrafficCapture:trafficReplayer"),
-                            project(":CreateSnapshot"),
+            def dependencies = []
+            dependencies += stageConsoleDependenciesFromLibraries(project as Project,
+                    List.of(project(":libraries:kafkaCommandLineFormatter")),
+                    projectName, "src/main/docker/${escapedProjectName}")
+            dependencies +=stageConsoleDependenciesFromApplications(project as Project,
```

Comment:
very small nit, so only bother if you're in this file anyways, but missing space after the `+=`